agent/listener/psapi.py

```diff
@@ -149,8 +149,20 @@ def get_memory_node():
     mem_swap_percent = RunnableNode('percent', method=lambda: (ps.swap_memory().percent, '%'))
     mem_swap_used = RunnableNode('used', method=lambda: (ps.swap_memory().used, 'B'))
     mem_swap_free = RunnableNode('free', method=lambda: (ps.swap_memory().free, 'B'))
+    node_children = [mem_swap_total, mem_swap_free, mem_swap_percent, mem_swap_used]
+
+    # Unix specific metrics ~ sorry Windows! :'(
+    if environment.SYSTEM != 'Windows':
```

Comment:
As descibed in the psutil [documentation](https://psutil.readthedocs.io/en/latest/#psutil.swap_memory) on windows sin and sout on Windows are always set to ```0```. I have excluded sin and sout for Windows systems.
@jomann09 How do you think about that? Should sin and sout available on Windows systems?